### PR TITLE
Feature/add user

### PR DESCRIPTION
**Pull request by dranidis**
# Scope: Users

- Add user functionality
-  User credentials

### CODE REVIEW

File: add_user.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+## Commands for adding a user.
```
Comment (city-student1) [inline add_user.txt]: Please add more detail.

**Review by city-student1 — CHANGES_REQUESTED** [review]
Some files are empty. Please add some code to them.

File: add_user.txt (hunk 1 above)
Comment (dranidis) [reply add_user.txt]: Detail is added

**Comment by dranidis** [comment]
Files are ready for review

**Review by city-student1 — APPROVED** [review]
Well done! Great job!

**Pull request merged**